tests/test_packages/test_protocols/test_fipa.py

```diff
@@ -303,3 +293,132 @@ def test_dialogues():
     dialogues.dialogue_stats.add_dialogue_endstate(FIPADialogue.EndState.DECLINED_CFP, is_self_initiated=False)
     assert dialogues.dialogue_stats.self_initiated == {FIPADialogue.EndState.SUCCESSFUL: 1, FIPADialogue.EndState.DECLINED_PROPOSE: 0, FIPADialogue.EndState.DECLINED_ACCEPT: 0, FIPADialogue.EndState.DECLINED_CFP: 0}
     assert dialogues.dialogue_stats.other_initiated == {FIPADialogue.EndState.SUCCESSFUL: 0, FIPADialogue.EndState.DECLINED_PROPOSE: 0, FIPADialogue.EndState.DECLINED_ACCEPT: 0, FIPADialogue.EndState.DECLINED_CFP: 1}
+    assert dialogues.dialogues_as_seller is not None
+    msg = FIPAMessage(message_id=0,
+                      dialogue_reference=(str(0), ''),
+                      target=1,
+                      performative=FIPAMessage.Performative.ACCEPT)
+    msg.counterparty = "opponent"
+    result = dialogues.is_permitted_for_new_dialogue(msg)
```

Comment:
This can be failing for several reasons, e.g. incomplete dialogue_reference, wrong performative etc. So you're testing multiple things ..